kernel/src/fs/device.rs

```diff
@@ -64,6 +64,12 @@ impl DeviceId {
     pub fn minor(&self) -> u32 {
         ((self.0 >> 12) & 0xffff_ff00 | self.0 & 0x0000_00ff) as u32
     }
+
+    pub fn to_dev_t(&self) -> u32 {
```

Comment:
```rust
/// Converts to the `dev_t` type in Linux.
///
/// Reference: <...???>.
```
Is there an authoritative source for the `dev_t` encoding? I could not find one.